app/api/tests/test_tools.py

```diff
@@ -1,6 +1,6 @@
 from fastapi.testclient import TestClient
 from main import app  
-from app.services.tool_registry import validate_inputs
+from services.tool_registry import validate_inputs
```

Comment:
Fetch these utility functions from the `app` folder.

Reply:
Fixed it